spec/lib/iiif_manifest/v3/manifest_factory_spec.rb

```diff
@@ -81,7 +81,11 @@ def display_image
     let(:json_result) { JSON.parse(subject.to_h.to_json) }
 
     it 'has a label' do
-      expect(result.label).to eq 'A good book'
+      result
```

Comment:
Is this call to `result` necessary here?

Reply:
that was added for debugging breakpoint, forgot to remove it